dspace-server-webapp/src/main/java/org/dspace/app/rest/ItemAddBundleController.java

```diff
@@ -166,9 +174,18 @@ public ItemRest updateLicenseForBundle(@PathVariable UUID uuid,
                     "but the new one will not be attached.");
         }
         List<Bundle> bundles = item.getBundles(Constants.CONTENT_BUNDLE_NAME);
+        String oldLicense = null;
         for (Bundle clarinBundle : bundles) {
             List<Bitstream> bitstreamList = clarinBundle.getBitstreams();
             for (Bitstream bundleBitstream : bitstreamList) {
+                if (Objects.isNull(oldLicense)) {
```

Comment:
I think oldLicense will be always null, why this `if` condition?

Reply:
no, it don't have to be because at line 186 we add there license name and we want to do it only one time I think